lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -43,8 +43,9 @@
 )
 from shoppingcart.pdf import PDFInvoice
 from student.models import (
-    CourseEnrollment, CourseEnrollmentAllowed, NonExistentCourseError
-)
+    CourseEnrollment, CourseEnrollmentAllowed, NonExistentCourseError,
+    ManualEnrollmentAudit, UN_ENROLL_TO_ENROLL, ENROLL_TO_UN_ENROLL, ALLOWED_TO_ENROLL_TO_UN_ENROLL, ENROLL_TO_ENROLL,
+    UN_ENROLLED_TO_ALLOWED_TO_ENROLL, UN_ENROLL_TO_UN_ENROLL, ALLOWED_TO_ENROLL_TO_ENROLL)
```

Comment:
while this doesn't violate pep8/pylint for line length, it seems a bit excessively long. Can you break it up a bit?